onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.h

```diff
@@ -4,6 +4,9 @@
 #pragma once
 
 namespace onnxruntime {
+
+class NodeUnit;
```

Comment:
To make the "op_builder.h" independently includable, we would have to use include the `onnxruntime_c_api.h`, since we will need the ORT_MUST_USE_RESULT

There was a PR last year by me trying to extract all these macroes from c_api, be it was never merged .

Reply:
I think ORT_MUST_USE_RESULT is not needed in many cases now that Status is [[nodiscard]]

Reply:
FYI, the general recommendation is to just include the needed headers
https://google.github.io/styleguide/cppguide.html#Forward_Declarations